modules/opm-cli-ref-index.adoc

```diff
@@ -5,7 +5,16 @@
 [id="opm-cli-ref-index_{context}"]
 = index
 
-Generate Operator index container images from pre-existing Operator bundles.
+Generate Operator index for SQLite database format container images from pre-existing Operator bundles.
+
+[IMPORTANT]
+====
+As of {product-title} 4.11, the default Red Hat-provided Operator catalog releases in the file-based catalog format. The default Red Hat-provided Operator catalogs for {product-title} 4.6 through 4.10 released in the SQLite database format.
+
+The `opm index` command, including the related subcommands and flags, was deprecated along with the SQLite database format and will be removed in a future release. The command is still supported and must be used for custom catalogs and catalogs that use the deprecated SQLite database format. 
```

Comment:
Good point. Noted. I will clarify this when I update the PR.

Reply:
The `opm registry` command/subcommands are also deprecated for the same reason.

There are also portions of even the _new_ commands that are deprecated:
- `opm render` currently supports rendering sqlite files and catalog images. That functionality is deprecated.
- `opm migrate` currently supports converting an sqlite file/image to FBC. That functionality is deprecated.

Basically any code in opm that deals with sqlite databases or images is deprecated.

Reply:
I will adjust the admonition so that it is a little broader for customers. I have created a story to see if we should tweak the `opm` CLI reference: https://issues.redhat.com/browse/OSDOCS-3871